lib/kubeclient/common.rb

```diff
@@ -156,7 +156,7 @@ def self.parse_definition(kind, name)
           method_names = [singular_underscores, singular_underscores + plural_suffix]
         else
           # Something weird, can't infer underscores for plural so just give them up
```

Comment:
is the comment still valid? it might be read as confusing along with code that actually adds underscores

Reply:
Pls let me know if the updated comment makes more sense or requires any improvements / being removed entirely.

Reply:
yes it does, thanks :)
I would say that "something weird" part is too generic and redundant, but it was here before, so I'm not insisting on taking it out.